content/en/tests/code_coverage.md

```diff
@@ -15,7 +15,7 @@ further_reading:
 ---
 
 {{< callout url="https://www.datadoghq.com/product-preview/code-coverage/" >}}
-Datadog is building a new dedicated Code Coverage product. Sign up for the Preview!
+This Test Optimization feature is being deprecated and replaced by a new dedicated <a href="https://www.datadoghq.com/product-preview/code-coverage/">Code Coverage</a> product. Sign up for the Preview!
```

Comment:
FYI: Ideally the inline link should point to the new Code Coverage docs page, but it's using the signup page as a temporary placeholder. The new Code Coverage page won't exist until after this PR is merged, so trying to link to it now would probably break the link checker in the docs build.

After we merge this PR, I can follow up with a quick PR to update the link URL here and on the JUnit setup page.